pkg/storaged/filesystem/filesystem.jsx

```diff
@@ -83,15 +83,9 @@ export function make_filesystem_card(next, backing_block, content_block, fstab_c
     const mismount_warning = check_mismounted_fsys(backing_block, content_block, fstab_config);
     const mounted = content_block && is_mounted(client, content_block);
 
-    let mp_text;
-    if (mount_point) {
-        mp_text = client.strip_mount_point_prefix(mount_point);
-        if (mp_text == false)
-            return null;
-        if (!mounted)
-            mp_text = mp_text + " " + _("(not mounted)");
-    } else
-        mp_text = _("(not mounted)");
+    const mp_text = mount_point_text(mount_point, mounted);
+    if (mp_text == null)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19824-20240108-104540-fcdf3177-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)